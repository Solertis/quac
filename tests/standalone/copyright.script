#!/bin/bash

# Print the name of each file that does not contain the copyright notice
# (below); the test then compares this against a list of exceptions (the
# .stdout file).
#
# Copyright (c) 2012-2013 Los Alamos National Security, LLC, and others

. ./environment.sh

cd ../..

C='Copyright (\(c\)|Â©) 2012-2013 Los Alamos National Security, LLC, and others'

for filename in $(git ls-files); do
    if (echo $filename | egrep -qv '(\.stdout$|\.stderr$)'); then
        egrep -iL "$C" $filename || true
    fi
done
