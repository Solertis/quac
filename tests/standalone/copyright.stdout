.gitignore
COPYRIGHT.rst
LICENSE
lib/disco/README
lib/geo/__init__.py
lib/qr/__init__.py
lib/unicodedata2.py
misc/halloween.xls
misc/tweet-volume.xls
sphinx/Makefile
sphinx/conf.py
tests/README
tests/twitter/README
tests/twitter/tweets/ids.txt
